### PR TITLE
Move asciidoc to proper python package structure

### PR DESCRIPTION
**Pull request by MasterOdin**
Start of work for #55. Haven't yet fixed the asciidocapi.py file yet nor hooked up the automated test suite, however this does layout the basis of how one might look at an asciidoc module.

Installation of it is done by doing: `python3 setup.py install`

Usage is then:
`asciidoc --help`
`a2x --help`

You can also use the module directly within the repository by doing:
`python3 asciidoc --help`
`python3 -m asciidoc.asciidoc --help`
`python3 -m asciidoc.a2x --help`

### CODE REVIEW

**Comment by MasterOdin** [comment]
I'd like to at least get the testasciidoc.py to work first before merging as that'd give me confidence that all parts of this are working as expected.

**Comment by elextr** [comment]
No problem, will hold off merging, just ping me when its ready.

**Comment by MasterOdin** [comment]
@elextr, I'd say this is ready to merge. The full teste suite now passes, and I was able to use asciidoc independently as well when installed/as a module to get the same result on a handful of testcases that I tried separate from the suite.

What would you think are the next steps on this?

File: asciidoc/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+"""asciidoc module"""
```
Comment (elextr) [inline asciidoc/__init__.py]: Lets standardize on having newlines

**Comment by elextr** [comment]
What happened to the CSSes that git(hub) seems to think are whole new files?

Otherwise it looks ok from inspection.

Note, ignore my post above about license, it may not be deliberate (given that COPYRIGHT says GPLv2 or later) but since `asciidoc.py` header does not say "or later" then effectively GPLv2 applies.

**Comment by MasterOdin** [comment]
I just copied them from the root into the resources folder while leaving the root in place, since I was originally unsure exactly how they were used. However, it looks like they're just always included directly into the outputted source so I believe I could remove the root and just then have them be considered a git mv.

**Comment by elextr** [comment]
Sorry for the delay, I have been out of the loop and didn't notice that the PR had been updated to make the changes into moves.  Am now catching up a bit.

**Pull request merged**